data/mods/Aftershock/items/gun/voltaic.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "id": "afs_electro_anomaly_rifle",
+    "type": "GUN",
+    "copy-from": "laser_rifle",
+    "name": { "str": "R-PROTON ENR" },
+    "description": "Within the solid barrel of the Raaz PROTON, an electrical anomaly is shredded and duplicated through magnetic stimulation and projected outwards.  Freed from containment, it will rapidly evaporate within a cloud of electric charge.",
```

Comment:
```suggestion
    "description": "The Raaz PROTON as a gun has earned itself a rather niche, but still noteworthy reputation within the class of Voltaics.  Within the solid barrel of it, a relatively common electric anomaly from the hyperspace age is shredded and duplicated through applications of both transistors and magnets before being expelled and once freed from containment, it will rapidly evaporate within a cloud of electric charge.",
```